lib/portlayer/exec/exec.go

```diff
@@ -76,12 +80,47 @@ func Init(ctx context.Context, sess *session.Session, source extraconfig.DataSou
 			Config.ResourcePool = o.ResourcePool
 		case *object.ResourcePool:
 			Config.ResourcePool = o
+			Config.VMGroupName = o.Name()
```

Comment:
Can you add a `// TODO` here reminding us to investigate whether all valid VM names are valid VM Group names? (I can't quickly find a reference that confirms that the length, allowed character set, etc. are all the same.)

Reply:
Can do - although as a note that's the name of the resource pool, not the VM.